### ISSUE TITLE
Is this method correct ?

### ISSUE THREAD

**Issue by ssizmaster** [brianfrankcooper/YCSB#1311]
Hi,

I would compare HBase and MySQL Server in terms of scalability by using YCSB on parallel mode.

For HBase, I installed HBase in standalone mode. Then, to scale out I add regionservers to my cluster.
For MySQL, I installed MySQL server in the first machine. Then, to scale out I install MySQL server in other machines of the cluster. 

For load and run phases, I run the commands in each machine(For MySQL and HBase ) and I aggregate the results thereafter.

Is this method correct to evaluate the scalability of MySQL server?

Please, I really need your help.

**Comment by busbey** [comment]
are the mysql servers running in some cluster mode? or are you assuming mysql sharding will be handled at the application layer?

are you running YCSB according to our [guidelines for running multiple YCSB clients](https://github.com/brianfrankcooper/YCSB/wiki/Running-a-Workload-in-Parallel)?

**Comment by ssizmaster** [comment]
Thank you for your response.

I assuming that mysql sharding will be handled at the application layer by using  the guidelines for running multiple YCSB clients.
Is this methode correct?

**Comment by busbey** [comment]
no, the parallel YCSB instances assume they are all talking to a single key space. For example, workloads that use Zipf distribution for operation requests will parallelize the inserts across all the mysql shards but their reads will very likely only hit a single shard.

You should configure MySQL in some cluster deployment if you want to test scaling mysql out across multiple servers. You should check MySQL documentation or forums for ways to do this.
